data/payload.go

```diff
@@ -17,18 +17,25 @@ type Payload struct {
 	SuspectedBinaries        []string
 	RepositoryMetadata       RepositoryMetadata
 	DependencyManifestsCount int
-
-	client *githubv4.Client
+	IsCodeRepo               bool
+	client                   *githubv4.Client
```

Comment:
```suggestion
	IsCodeRepo               bool

	client                   *githubv4.Client
```
```suggestion
	IsCodeRepo               bool

	client                   *githubv4.Client
```

nitpicking... I'd like to keep public and private vars separated by a line, just for easy code skimming